testing/test_webinterface.py

```diff
@@ -74,7 +76,7 @@ def test_webinterface_with_activate(webinterface_service):
     app_log = webinterface_service.read_log()
 
     # Verify everything is in place.
-    assert "Activating setup id=1" in app_log
+    assert "Activating setup id=" in app_log
```

Comment:
Why was this `id` lost?

Reply:
Not lost. Just counted up in DB. Was to lazy to clean DB every run. Can be 0 if DB is new.

Reply:
Please keep resetting the database on every test function ;].